README.md

```diff
@@ -146,17 +163,17 @@ sub-ON<subject>
 
 Definitions:
 
-- `<subject>` = subject number
+- `<subject>` = participant ID
 - `<taskname>` = task name: `airpuff`, `artifact`, `gonogo`, `haririhammer`, `movie`, `oddball`, `sternberg`
 - `<desc>` = placeholder for acquisition label for a given suffix
-- `<direction>` = flipped, unflipped
+- `<direction>` = `flipped`, `unflipped`, `forward`, `reverse`
 - `<index>` = run number/index
 - `<suffix>` = placeholder to indicate the scan type
-  - `T1w`: `<desc>` = `fspgr`, `mprage`, `fse`, `highreshippo`
-  - `T2w`: `<desc>` = `abcdcube`, `cube`, `frfse`
-  - `FLAIR`: `<desc>` = `adni2d`, `2d`, `3d`, `t2`
+  - `T1w`: `<desc>` = `MPRAGE`, `FSPGR`, `HighResHippo`
+  - `T2w`: `<desc>` = `CUBE`
+  - `FLAIR`: `<desc>` = `2dADNI2`, `2d`, `3d`
   - `epi`: `<desc>` = `dwib1000`, `dwi`, `resting`
-  - `T2star`
+  - `T2starw`
   - `bold`
   - `meg`
   - `asl`
```

Comment:
This is all great!  Do we need to update the BIDS tree graphic at all in the README due to our updates?

Reply:
This is now done.